plugins/ImageGraph/API.php

```diff
@@ -139,7 +139,7 @@ public function get(
         }
 
         $useUnicodeFont = array(
-            'am', 'ar', 'el', 'fa', 'fi', 'he', 'ja', 'ka', 'ko', 'te', 'th', 'zh-cn', 'zh-tw',
+            'am', 'ar', 'el', 'fa', 'fi', 'fr', 'he', 'ja', 'ka', 'ko', 'te', 'th', 'zh-cn', 'zh-tw',
```

Comment:
I think we should either use the unifont for all languages if present, or only for those that we know using characters not supported by the default font.

Personally I would for now remove French from that list again. We are adding a quick fix for the special space character, so this should also work without unifont.

Imho we should provide some more possibilities around configuring fonts that should be used for reports, but that is something fully out of scope here.